src/modules/booking/containers/my-bookings-content/my-bookings-content.tsx

```diff
@@ -1,58 +1,63 @@
 'use client';
 import { type FC } from 'react';
-import { faker } from '@faker-js/faker';
 // components
 import { BookingsList } from '@/modules/booking/components/bookings-list';
 import { Calendar } from '@/modules/schedule/components/calendar';
 // hooks
 import { useMyBookingsTab } from '@/modules/booking/hooks/use-my-bookings-tab';
-
-import type {
-  MyBookingsContentProps,
-  TabContentMap,
-} from './my-bookings-content.interface';
+// utils
+import { trpc } from '@/modules/core/utils/trpc.utils';
+// type
+import type { MyBookingsContentProps } from './my-bookings-content.interface';
+// style
 import styles from './my-bookings-content.module.scss';
 
-const createEventsListMock = (num: number) =>
-  [...new Array(num)].map(() => ({
-    id: faker.string.uuid(),
-    name: faker.person.fullName(),
-    serviceName: 'evening makeup',
-    date: faker.date.future().toString(),
-  }));
+import { type Booking, type ServiceOnProfessional } from '@prisma/client';
 
-const upcomingEvents = createEventsListMock(1);
-const pastEvents = createEventsListMock(2);
+export const MyBookingsContent: FC<MyBookingsContentProps> = () => {
+  const { data: me } = trpc.user.me.useQuery({ expand: ['professional'] });
+  // query
+  const { data: upcomingEvents } = trpc.booking.list.useQuery({
+    expand: ['serviceProfessional'],
+    sortField: 'date',
+    sortDirection: 'desc',
+    professionalId: me?.id,
+  });
 
-const tabContentMap: TabContentMap = {
-  list: {
-    content: (
-      <BookingsList
-        groups={[
-          {
-            id: 'upcomming',
-            title: 'Upcomming',
-            cardsVariant: 'light',
-            list: upcomingEvents,
-          },
-          {
-            id: 'past',
-            title: 'Past events',
-            cardsVariant: 'light',
-            list: pastEvents,
-          },
-        ]}
-      />
-    ),
-  },
-  calendar: {
-    content: <Calendar />,
-  },
-};
+  const { data: pastEvents } = trpc.booking.list.useQuery({
+    expand: ['serviceProfessional'],
+    sortField: 'date',
+    sortDirection: 'asc',
+    professionalId: me?.id,
+  });
 
-export const MyBookingsContent: FC<MyBookingsContentProps> = () => {
   const { activeTab } = useMyBookingsTab();
-  const { content } = tabContentMap[activeTab];
 
-  return <div className={styles.root}>{content}</div>;
+  return (
+    <div className={styles.root}>
+      {activeTab === 'list' && (
+        <BookingsList
+          groups={[
+            {
+              id: 'upcomming',
+              title: 'Upcomming',
+              cardsVariant: 'light',
+              list: upcomingEvents as (Booking & {
```

Comment:
Try not to use as or at least move this to some separate type instead of writing it right here.